Refit/targets/refit.props

```
@@ -4,4 +4,17 @@
     <CompilerVisibleProperty Include="RefitInternalNamespace" />
   </ItemGroup>
 
+  <Choose>
+    <When Condition="'$(VisualStudioVersion)' >= '17.0' AND '$(DesignTimeBuild)' == 'True'">
```

Check out https://github.com/dotnet/msbuild/pull/4911. Sorry - In my previous reply I didn't have time to look it up.

```suggestion
    <When Condition="$([MSBuild]::VersionGreaterThanOrEquals($(VisualStudioVersion), '17.0')) AND '$(DesignTimeBuild)' == 'True'">
```